string pick_obj
---
bool success
